wix/yoshi
# Support Tree Shaking and Dynamic Imports in Typescript

### 🔦 Summary
This change adds support for both Tree Shaking and Dynamic Imports in Typescript.
This is a breaking change since we change the module definition to `esnext`.
Fixes #343 #344 


### 🗺️ Test Plan
- Check that webpack tree shakes unused imports and transpiles the used ones to commonjs
- Check that commonjs is enforced for tests and server code
